ModelVale/ModelVale/CustomComponents/Datatypes/ModelLabel.m

```diff
@@ -6,19 +6,21 @@
 //
 
 #import "ModelLabel.h"
+#import "ModelData.h"
 #import "TestTrainEnum.h"
+#import "Parse/Parse.h"
 
 @implementation ModelLabel
 
-- (ModelLabel*) initEmptyLabel: (NSString*)label testTrainType: (testTrain)testTrainType {
+- (ModelLabel*) initEmptyLabel: (NSString*)label testTrainType: (NSString*)testTrainType {
```

Comment:
I mentioned this in a previous PR but make sure to properly override the NSObject initializer https://medium.com/@MarkEdwardMurray/objective-c-class-initializers-a-k-a-constructors-94dcd6d2ffd7